### PR TITLE
Cleanup and Fixes

### PR DESCRIPTION
**Pull request by StarArawn**
Fixes: #138 
Fixes: #136

### CODE REVIEW

**Comment by turboMaCk** [comment]
Can speak only about changes in examples but what I see makes sense! thanks.

File: src/map_query.rs [hunk 1]
```diff
@@ -422,6 +422,7 @@ impl<'w, 's> MapQuery<'w, 's> {
                         grid_size.x,
                         grid_size.y,
                     );
+                    dbg!(grid_size, layer_size_in_tiles, map_size, map_pos, center);
```
Comment (subnomo) [inline src/map_query.rs:425]: I've created a PR to revert this change: #152.
